pysr/export_jax.py

```diff
@@ -192,9 +192,6 @@ def sympy2jax(expression, symbols_in, selection=None, extra_jax_mappings=None):
     )
     hash_string = "A_" + str(abs(hash(str(expression) + str(symbols_in))))
     text = f"def {hash_string}(X, parameters):\n"
-    if selection is not None:
```

Comment:
Why was this part removed? It seems to make the JAX output no longer work when feature selection is turned on, e.g.,

```python
import numpy as np
import pandas as pd
from pysr import PySRRegressor
from functools import partial

X = pd.DataFrame({f"k{i}": np.random.randn(1000) for i in range(10, 21)})
y = X["k15"] ** 2 + np.cos(X["k20"])

model = PySRRegressor(unary_operators=["cos"], select_k_features=3, early_stop_condition=1e-5)
model.fit(X.values, y.values)
f, parameters = model.jax().values()

np_prediction = model.predict
jax_prediction = partial(f, parameters=parameters)

np_output = np_prediction(X.values)
jax_output = jax_prediction(X.values)

np_loss = np.average(np_output - y.values)
jax_loss = np.average(jax_output - y)
```

`np_loss` is zero but `jax_loss` seems to be much larger - indicating the input array is not being masked with the selection.

Reply:
Okay I reverted this and it seems to work!